Corona-Warn-App/src/main/res/layout/contact_diary_add_location_fragment.xml

```diff
@@ -0,0 +1,132 @@
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/contact_diary_add_location_close_button">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/contact_diary_add_location_name_input_edit_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:imeOptions="actionNext"
+                android:inputType="textCapWords" />
+
+        </com.google.android.material.textfield.TextInputLayout>
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/contact_diary_add_location_phone_input_layout"
+            style="@style/TextInputLayoutTheme"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_tiny"
+            android:hint="@string/contact_diary_add_text_input_phone_hint"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/contact_diary_add_location_name_input_layout">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/contact_diary_add_location_phone_input_edit_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:imeOptions="actionNext"
+                android:inputType="phone" />
+
+        </com.google.android.material.textfield.TextInputLayout>
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/contact_diary_add_location_email_input_layout"
+            style="@style/TextInputLayoutTheme"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_tiny"
+            android:hint="@string/contact_diary_add_text_input_email_hint"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/contact_diary_add_location_phone_input_layout">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/contact_diary_add_location_email_input_edit_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:imeOptions="actionDone"
+                android:inputType="textEmailAddress" />
+
+        </com.google.android.material.textfield.TextInputLayout>
+
+        <android.widget.Button
```

Comment:
Could be just `<Button` right?

Reply:
Yep, I this was from the old layout and I didn't catch it.